kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -268,4 +270,13 @@ private static String slug(LocalRef<?> ref) {
         return ref.getKind().toLowerCase(Locale.ROOT) + groupString + "/" + name;
     }
 
+    public static boolean isReconciled(VirtualKafkaCluster cluster) {
```

Comment:
To me this is more like `isStatusCurrent`. Also is there any need for the `Function` version right now?

Reply:
I guess if we were using "stale" for the opposite sense then this should be `isStatusFresh`.

Reply:
Function version is just there anticipating that we're going to implement the same for the other referents soon